### PR TITLE
Add OzBargain

### PR DESCRIPTION
**Pull request by ColinHebert**
ozbargain.com.au [rank 3,226](https://www.alexa.com/siteinfo/ozbargain.com.au) supports TOTP, add details about it.

### CODE REVIEW

**Comment by ColinHebert** [comment]
Of course, there is an outage right when I submit my pull-request.
Hopefully it'll be back online when the PR gets reviewed, if not, the fact that this site is #50 in Alexa ranking for Australia might give some weight to the fact that this is a legitimate request.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I see you've used some kind of machine learning to convert a PNG image to the SVG format. This doesn't work.
If you can't find a real SVG image, follow the instructions in our [contribution guidelines](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines)
 on how to use PNG images as icons.

**Comment by ColinHebert** [comment]
@Carlgo11 should be okay now.

**Review by Carlgo11 — APPROVED** [review]

**Comment by ColinHebert** [comment]
Found a better/smaller SVG

File: img/o/ozbargain.com.au.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 98 98" xmlns="http://www.w3.org/2000/svg"><g transform="translate(0 -954)"><path d="M0 954h98v98H0z" fill="#cd6702"/><rect x="13" y="967.4" width="72" height="72" rx="10" ry="10" fill="none" stroke="#fff" stroke-width="10"/><text x="20" y="1028.2" fill="#fff" font-family="Impact" font-size="15.7" font-weight="400" letter-spacing="0" word-spacing="0" style="line-height:125%" xml:space="preserve"><tspan x="20" y="1028.2" font-size="62.8">Oz</tspan></text></g></svg>
```
Comment (phallobst) [inline img/o/ozbargain.com.au.svg]: Using text in an SVG icon may not render properly on every device - better use a path for the letters.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 98 98"><path fill="#cd6702" d="M0 0h98v98H0Z"/><rect width="72" height="72" x="13" y="13.4" fill="none" stroke="#fff" stroke-width="10" rx="10"/><path fill="#fff" d="M52 53.6q0 7.5-.3 10.6-.3 3.1-2.2 5.7-1.8 2.6-5 4-3.2 1.3-7.4 1.3-4 0-7.1-1.2-3.2-1.4-5.1-4-2-2.6-2.3-5.6-.4-3.1-.4-10.8v-8.5q0-7.5.3-10.5.4-3.2 2.3-5.8 1.8-2.5 5-4 3.1-1.3 7.3-1.3 4 0 7.2 1.3t5.1 4q2 2.5 2.3 5.6.4 3 .4 10.7zM39.3 37.3q0-3.5-.4-4.4-.4-1-1.6-1-1 0-1.6.8-.5.8-.5 4.6v23.2q0 4.3.4 5.3.3 1 1.6 1t1.7-1.1q.4-1.2.4-5.6zm36.5-3.8v9L66.7 66h9.1v8.2H54.7v-8.5l9.3-24h-8.4v-8.2z" letter-spacing="0" word-spacing="0"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: img/o/ozbargain.com.au.svg (hunk 1 above)
Comment (ColinHebert) [reply img/o/ozbargain.com.au.svg]: I thought so too, but then again that's what's provided on the original SVG: https://www.ozbargain.com.au/wiki/help:api

I've used https://files.delvu.com/images/ozbargain/logo/Square%20Flat.svg and removed the two broken filters (`filter:url(#filter4454)` and `filter:url(#filter4482)`

**Review by phallobst — APPROVED** [review]

**Pull request merged**